.github/workflows/pr_push.yml

```diff
@@ -48,6 +48,31 @@ jobs:
         - name: Build
           run: >
             cmake --build ${{github.workspace}}/build --config Release -j ${{matrix.nproc}}
+
+    CodeStyle:
+      name: Coding style
+      runs-on: ubuntu-latest
+
+      steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install apt packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake clang-format-15
+
+      - name: Configure CMake
+        run: >
+          cmake
+          -B ${{github.workspace}}/build
+          -DUMF_FORMAT_CODE_STYLE=ON
```

Comment:
The coding style of the whole repository should be checked, so all components turned off by default should be also enabled:
- UMF_BUILD_LIBUMF_POOL_DISJOINT
- UMF_BUILD_LIBUMF_POOL_JEMALLOC
- UMF_BUILD_LIBUMF_POOL_SCALABLE
- UMF_BUILD_BENCHMARKS

Reply:
does it make any difference...? we use glob pattern to find files in repo for scanning

Reply:
Right